tests/cmd/fault-trigger/main.go

```diff
@@ -43,7 +45,7 @@ func main() {
 	logs.InitLogs()
 	defer logs.FlushLogs()
 
-	mgr := manager.NewManager()
+	mgr := manager.NewManager(vmManager)
```

Comment:
better not to allow invalid `--vm-manager` value, e.g.

```
var vmMgr VMManager
if vmManager == "qm" {
   vmMgr = NewQMManager()
} else if vmManager == "virsh" {
  vmMgr = NewVirshManager()
} else {
   // fatal error
}

mgr := manager.NewManager(vmMgr)
```

if some users configured an invalid value, but our program still works, this will confuse people because they don't know what virtual manager we use from the command-line flags unless they know implementation details